src/Components/AboutUsComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+
+
+function AboutUsComponent() {
+  return (
+    <div className='  large:space-x-10  medium:space-x-10   '> 
+<div className='small:text-center medium:text-center large:text-center'>
+<h1 className='font-semibold text-5xl  ml-56 small:ml-8	medium:ml-8 large:ml-8  ' style={{color:"#16a34a"}}> About <span className="text-secondary  ">Us?</span></h1>
```

Comment:
Let's stick to tailwind CSS. You can add the text color to the same color by adding `text-green-600` to the class of the h1.